pkg/cpio/mknod_freebsd.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cpio
+
+import (
+	"syscall"
+)
+
+func Mknod(path string, mode uint32, dev int) (err error) {
```

Comment:
make this lower case unless you really want it exported outside cpio. If you want it exported you'll need a comment but ... I doubt you want it exported.